Assets/Scripts/Statics/Difficulty.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Difficulty
```

Comment:
Is this class a singleton class by default?

Reply:
No it is not and doesn't need to be one, it acts as a functional static class, namely just to calculate how enemies scale their health and attack